app/services/resource_recommendation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ResourceRecommendation
```

Comment:
Not a blocker by any means, but what if we pluralized this class name to `ResourceRecommendations` to make it clear that we'll be getting multiple recommendations?